metrics.go

```diff
@@ -32,23 +33,31 @@ type ConsumerMetrics struct {
 	numWorkerManagersGauge metrics.Gauge
 	activeWorkersCounter   metrics.Counter
 	pendingWMsTasksCounter metrics.Counter
+	taskTimeoutCounter     metrics.Counter
 	wmsBatchDurationTimer  metrics.Timer
 	wmsIdleTimer           metrics.Timer
 }
 
-func newConsumerMetrics(consumerName string) *ConsumerMetrics {
+func newConsumerMetrics(consumerName, prefix string) *ConsumerMetrics {
 	kafkaMetrics := &ConsumerMetrics{
-		registry: metrics.NewRegistry(),
+		registry: metrics.DefaultRegistry,
```

Comment:
https://github.com/stealthly/go_kafka_client/pull/126 also contained this change and I merged it. Maybe will change this line if necessary in a separate PR